core/src/rpc.rs

```diff
@@ -730,6 +746,13 @@ pub trait RpcSol {
         commitment: Option<CommitmentConfig>,
     ) -> Result<u64>;
 
+    #[rpc(meta, name = "getLargestAccounts")]
+    fn get_largest_accounts(
```

Comment:
wdyt about adding an input to allow the user to select between circulating and non-circulating largest accounts?

Reply:
You want that in this pr, even though the ability to distinguish circ/non-circ doesn't exist yet?

Reply:
Na, adding it later when we add that support works fine.    I was just thinking about the RPC API and the docs, and defining the current (default) behaviour in them

Reply:
To that point... docs, will add :)

Reply:
hehe, did you see how I did that?  ;)

Reply:
✨